code/modules/mob/living/carbon/human/species_types/lizardpeople.dm

```diff
@@ -89,6 +90,12 @@
 					H.emote("wag")
 				if(-1)
 					stop_wagging_tail(H)
+	if(!regrowtimer && !("waggingtail_lizard" in mutant_bodyparts) && !("tail_lizard" in mutant_bodyparts))
+		regrowtimer = addtimer(CALLBACK(src, .proc/regrow_tail), 30 MINUTES, TIMER_UNIQUE)
+
+/datum/species/lizard/proc/regrow_tail()
+	if(!("waggingtail_lizard" in mutant_bodyparts) && !("tail_lizard" in mutant_bodyparts))
+		mutant_bodyparts |= "tail_lizard"
```

Comment:
maybe add a to_chat for this? Feels like there isnt enoug feedback.

Reply:
a visible message would be better imo